packages/neo-one-node-vm/src/opcodes.ts

```diff
@@ -473,6 +473,31 @@ const functionCallIsolated = ({ name }: { readonly name: OpCode }): OpCreate =>
   },
 });
 
+const newArrayOrStruct = ({ name }: { readonly name: 'NEWARRAY' | 'NEWSTRUCT' }) =>
```

Comment:
Curious why this needs to be deconstructed like this? Would:
```
const newArrayOrStruct = (name: 'NEWARRAY' | 'NEWSTRUCT') => ...
```
not work? Maybe its typing issue.